pkgs/development/haskell-modules/hackage-packages.nix

```diff
@@ -119541,8 +119541,6 @@ self: {
        testToolDepends = [ hspec-discover ];
        description = "A modern format for Haskell packages";
        license = stdenv.lib.licenses.mit;
-       hydraPlatforms = stdenv.lib.platforms.none;
-       broken = true;
```

Comment:
This might fix eval, but is it actually broken?

Reply:
See https://github.com/NixOS/nixpkgs/pull/66493#issuecomment-520264858

Reply:
Why not keep the `hydraPlatforms`?

Reply:
Explained in https://github.com/NixOS/nixpkgs/pull/66493#issuecomment-520264858

Reply:
I just did what a package not marked as broken would generate